index.html

```diff
@@ -4,49 +4,50 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
     <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="styles.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
-    <title>Portfolio</title>
+    <title> Porfolio</title>
 </head>
 <body>
-  <section class="first-page">
-    <div class="main">
-      <nav class="navbar">
-        <div class="logo"><a href="#">Eros</a></div>
-        <div class="menu"><i class='bx bx-menu'></i></box-icon></div>
-    </nav>
-    <div class="headline">
-    <div class="description">
-      <h1 class="name"> I am Aime
-          Glad to see you!</h1>
-         
-      <p class="text"> I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-           </p>
-      </div>
-      <div class="contact">
-        <button>
-
-          <h2 class="media">LET'S CONNECT:</h2>
-        
-        </button>
-        <ul>
-         
-          <li><a href="https://twitter.com/AimeSebagabo"><i class='bx bxl-twitter'></i></a></li>
 
-          <li><a href="https://www.linkedin.com/in/jean-aime-sebagabo-279aa2207/"></a><i class='bx bxl-linkedin'></i></a></li>
-      
-           <li><a href="http://yourgmailhandle"><i class='bx bxl-gmail'></i></a></li>
-      
-           <li><a href="https://github.com/Hazard10-eng"><i class='bx bxl-github'></i></a</a</li>
-        </ul>
+    <section class="mobile-portfolio">
+    <section class="first-page">
+        <div class="main">
+          <nav class="navbar">
+            <div class="logo"><a href="#">Eros</a></div>
+            <div class="menu"><i class='bx bx-menu'></i></div>
+        </nav>
+        <div class="headline">
+        <div class="description">
+          <h2 class="name"> I am Aime
+              Glad to see you!</h2>
+             
```

Comment:
- [ ] Great job adding this description title :clap:  , however its font weight should stick to the FIGMA design. Kindly make sure you change that :+1: 

Your Design

![Screenshot from 2023-01-12 16-25-29](https://user-images.githubusercontent.com/80458199/212093197-54f6fa5f-ac50-4b0c-9044-1eba394320fd.png)


Figma design

![Screenshot from 2023-01-12 16-25-51](https://user-images.githubusercontent.com/80458199/212093289-fb545389-bdd6-47c1-9604-c0a9de46e7c1.png)